### DISCUSSION TITLE
Inactivity of  gatsbyjs / gatsby-de

### DISCUSSION THREAD

**Discussion by psjnek** [gatsbyjs/gatsby#37235] [Help] [answered]
I noticed that gatsby-de is owned by seemingly very inactive GitHub accounts. Some of the PR's there are years old and are still open.
I would offer to become a maintainer myself to revive the repo. Let's find a solution to that problem together.

Thanks in advance
WebSnke

**Comment by LekoArts** [comment, marked as answer]
Hi! The translation effort itself is stopped so there aren't any plans to incorporate/use the translations on our official site. Appreciate it though that you want to contribute! We have some guidance on where help is appreciated: https://www.gatsbyjs.com/contributing#how-to-contribute

**Reply by psjnek** [reply]
Ok, thanks.
